Index.html

```diff
@@ -0,0 +1,127 @@
+</head>
+<body>
+  <header>
+    <div>
+      <nav>
+        <ul>
+          <li><a href="#nowhere" title="Lorum ipsum dolor sit amet">Lorem</a></li>
+          <li><a href="#nowhere" title="Aliquam tincidunt mauris eu risus">Aliquam</a></li>
+          <li><a href="#nowhere" title="Morbi in sem quis dui placerat ornare">Morbi</a></li>
+          <li><a href="#nowhere" title="Praesent dapibus, neque id cursus faucibus">Praesent</a></li>
+          <li><a href="#nowhere" title="Pellentesque fermentum dolor">Pellentesque</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <div>
+    <section id="home">
+      <div>
+        <img src="" alt="profile picture">
+      </div>
+      <div>
+        <h1>
+          Miguel Ezequiel Solano Abreu
+        </h1>
+      </div>
+      <div>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Autem sint voluptatum laboriosam esse hic obcaecati, numquam animi facere pariatur, magnam labore omnis sapiente odio sunt aut qui nisi possimus error.
+      </div>
+    </section>
+    <section id="professional">
+      <div>
+        <h3>
+          Professional
+        </h3>
+      </div>
+      <div>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Rem quasi dolorem, accusantium nostrum, porro corrupti illo non cupiditate! Aliquid dolorum suscipit libero, laborum repudiandae quod earum! Minus deleniti sed aliquid.
+      </div>
+      <div>
+        <span>Lorem 1</span>
+      </div>
+      <div>
+        <span>Lorem 2</span>
+      </div>
+      <div>
+        <span>Lorem 3</span>
+      </div>
+      <div>
+        <span>Lorem 4</span>
+      </div>
+    </section>
+    <section id="experince">
+      <div>
+        <h3>
+          Experience
+        </h3>
+      </div>
+      <div>
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nihil excepturi obcaecati laborum iusto quia debitis aliquam recusandae, ex hic. Saepe sapiente est possimus libero molestiae ipsa consequatur praesentium autem soluta.
+      </div>
+    </section>
+    <section id="awards_&_interest">
+      <div>
+        <h3>
+          Awards
+        </h3>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. 
+        </p>
+      </div>
+      <div>
+        <h3>
+          Interest
+        </h3>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. 
+        </p>
+      </div>
+    </section>
+    <section id="contact">
+      <div>
+        <h3>
+          Contact
+        </h3>
+      </div>
+      <div>
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Consectetur optio praesentium autem sit quam modi magni delectus! Aut doloremque facilis debitis.
+        </p>
+      </div>
+      <div>
+        <form action="#" method="post">
+          <div>
+            <label for="name">Text Input:</label>
+            <input type="text" name="name" id="name" value="" tabindex="1" />
```

Comment:
⚠️ for attribute of labels should point to unique values of Id. In this case id `name` is been repeated many times.